Plugins/Public/base_plugin/Main.cpp

```diff
@@ -3030,6 +3040,9 @@ void AddFactoryRecipeToMaps(const RECIPE& recipe)
 	recipeMap[recipe.nickname] = recipe;
 	recipeCraftTypeNumberMap[recipe.craft_type][recipe.shortcut_number] = recipe;
 	recipeCraftTypeNameMap[recipe.craft_type][recipeNameKey] = recipe;
+	if (recipe.unlocked_by != 0) {
```

Comment:
opening bracket in a separate line please, also please remove the != 0, it's redundant.